### PR TITLE
Update Minimum-Effort Tacit Coordination Game.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

File: tasks/Minimum-Effort Tacit Coordination Game.md [hunk 1]
```diff
@@ -35,7 +40,21 @@ Treatment C: In treatment C, group size was reduced to two people who were stran
 
 
 ## Instructions
-Authors no longer have instructions
```
Comment (markwhiting) [inline tasks/Minimum-Effort Tacit Coordination Game.md:38]: So, to be clear, these are from one of the documents linked above, right?

File: tasks/Minimum-Effort Tacit Coordination Game.md [hunk 2]
```diff
@@ -35,7 +40,21 @@ Treatment C: In treatment C, group size was reduced to two people who were stran
...
+
+You are provided with a table that tells you the potential payoffs you may receive. Please look at the table now. The earnings in each period may be found by looking across from the value you choose on the left-hand side of the table and down from the median value chosen from the top of the table. For example, if you choose a 4 and the median value chosen is a 3, you earn 85 cents that period.
+
+![image](https://user-images.githubusercontent.com/78745728/126657420-9ae8b3f1-77a2-48f4-af8b-bc63a69976c6.png)
+
+
```
Comment (markwhiting) [inline tasks/Minimum-Effort Tacit Coordination Game.md:51]:
```suggestion
```

Comment (markwhiting) [reply tasks/Minimum-Effort Tacit Coordination Game.md:51]: markdown optimizes whitespace, so I think more new lines after one never makes a difference.

**Review by markwhiting — APPROVED** [review]
Approved, the minor change I added doesn't matter, but feel free to integrate.

File: tasks/Minimum-Effort Tacit Coordination Game.md (hunk 1 above)
Comment (sarikasu) [reply tasks/Minimum-Effort Tacit Coordination Game.md:38]: Correct- the authors included this source in the footnotes of the main paper- I missed that the first time around.

**Review by markwhiting — APPROVED** [review]

**Pull request merged**